src/jdk.jfr/share/conf/jfr/default.jfc

```diff
@@ -739,6 +739,12 @@
       <setting name="threshold" control="file-threshold">20 ms</setting>
     </event>
 
+    <event name="jdk.Selection">
+      <setting name="enabled">true</setting>
+      <setting name="stackTrace">true</setting>
+      <setting name="threshold">20 ms</setting>
```

Comment:
The default threshold is 20ms but that means the default won't emit en event when spinning. If this event is used for performance and troubleshooting purposes then maybe we should emit an event when select returns 0 and not emit an event when selectNow returns 0 (selectNow is used to flush channels from the Selector so returning 0 is normal and not interesting).